src/constants/wallet.ts

```diff
@@ -53,7 +53,15 @@ export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
     description: 'Use Coinbase Wallet app on mobile device',
     href: null,
     color: '#315CF5',
+  },
```

Comment:
why are we modifying an original file from Uniswap?

Reply:
I am restoring it, it was like this, I had modified it in [another PR](https://github.com/gnosis/cowswap/pull/2166) where I thought it was custom.

Reply:
👌